docs/application/dotnet/guides/user-interface/nui/absolute-layout.md

```diff
@@ -0,0 +1,182 @@
+<style>
+    .tabcontent img {
+        border: 1px solid #555;
+        max-width: 100% !important;
+        max-height: 100%;
+    }
+</style>
+
+# Absolute Layout
+
+`AbsoluteLayout` allows explicit positioning of the child components using explicit values. The positions are defined from the top left corner of the layout and can be set using the `View.Positions` property.
+
+| Property | Type | Description |
+| --- | --- | --- |
+| `Position` | float, float, float | Sets the position of the component in the layout. |
+| `Position2D` | int, int | Sets the position of the component in the layout for X and Y axis.  |
+| `PositionX` | float | Sets the position of the component in the layout X axis. |
+| `PositionY` | float | Sets the position of the component in the layout Y axis. |
+
+## Position views with absolute layout
+
+![AbsoluteLayoutExample](./media/Absolute-Layout-Example.png)
+
+The following example shows how to use position views with the `AbsoluteLayout`:
+
+<div id="TabSection1">
+    <div class="sampletab " id="ProjectCreateTab">
+        <button id="AbsoluteLayout-Example-CSharp" class="tablinks " onclick="openTabSection(event, 'AbsoluteLayout-Example-CSharp', 'TabSection1') ">C#</button>
+        <button id="AbsoluteLayout-Example-Xaml" class="tablinks " onclick="openTabSection(event, 'AbsoluteLayout-Example-Xaml', 'TabSection1') ">Xaml</button>
+    </div>
+    <div id="AbsoluteLayout-Example-CSharp" class="tabcontent">
+        <table>
+            <tbody>
+                <tr>
+<span style="display:block">
+
+```csharp
+View absoluteLayoutView = new View
+{
+    WidthSpecification = LayoutParamPolicies.MatchParent,
+    HeightSpecification = LayoutParamPolicies.MatchParent,
+    BackgroundColor = Color.White,
+};
+
+absoluteLayoutView.Layout = new AbsoluteLayout();
+
+TextLabel header = new TextLabel
+{
+    Text = "Stylish Header",
+    PointSize = 25,
+    Position2D = new Position2D(290, 270)
+};
+
+View redView = new View
+{
+    BackgroundColor = Color.Red,
+    Position2D = new Position2D(232, 220),
+    WidthSpecification = 800,
+    HeightSpecification = 5
+};
+
+View greenView = new View
+{
+    BackgroundColor = Color.Green,
+    Position2D = new Position2D(232, 208),
```

Comment:
Just a question.

```
    BackgroundColor = Color.Red,
    Position2D = new Position2D(232, 220),
```

```
    BackgroundColor = Color.Green,
    Position2D = new Position2D(232, 208),
```

According to the code, doesn't the green come above the red?

It seems the order should be green (208) -> red (220) -> text (270) in Y axis
But in Absolute-Layout-Example.png, red comes above green.

Reply:
Right, i updated sample code.